app/models/organisation.rb

```diff
@@ -83,6 +83,10 @@ def all_vacancies
     Vacancy.in_organisation_ids(all_organisation_ids)
   end
 
+  def all_active_vacancies
+    Vacancy.in_organisation_ids(all_organisation_ids).active_in_current_academic_year
+  end
+
```

Comment:
don''t think this is the right solution - the issue IMHO is that we are somehow counting closed vacancies. I think we need some more tests around this area to show where things are going wrong?